install/kubernetes/helm/subcharts/nodeagent/values.yaml

```diff
@@ -6,3 +6,5 @@ image: node-agent-k8s
 env:
   CA_PROVIDER: ""
   CA_ADDR: ""
+  Trust_Domain: ""
```

Comment:
Shouldn't that be taken from https://github.com/istio/istio/blob/de3ce8ea77571bcf3c1e13376a138fb1416f19f9/install/kubernetes/helm/istio/values.yaml#L335?

Reply:
I was thinking about that, do you know how to refer in values.yaml ? otherwise I have to set this env variable from https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/nodeagent/templates/daemonset.yaml#L36

Reply:
Citadel is referring to it as `.Values.trustDomain` in https://github.com/istio/istio/blob/de3ce8ea77571bcf3c1e13376a138fb1416f19f9/install/kubernetes/helm/subcharts/security/templates/deployment.yaml#L51-L53, although that is referring to https://github.com/istio/istio/blob/de3ce8ea77571bcf3c1e13376a138fb1416f19f9/install/kubernetes/helm/subcharts/security/values.yaml#L8 instead of the global values.yaml...

@c0d1ngm0nk3y @axel7born shouldn't there only be one place to configure trustDomain?

Reply:
`.Values.global.trustDomain` should give you the value from overall istio values.yaml

Reply:
there should be only one place to set it, I'm going to add it to global configmap after https://github.com/istio/api/pull/697 get in; then both nodeagent and cidatel should refer value in the global confgmap.